src/display.hpp

```diff
@@ -72,6 +72,18 @@ namespace wb {
 #include <memory>
 #include <vector>
 
+struct submerge_data
```

Comment:
Since these are public members they shouldn't have trailing underscores. Also, can remove the default ctor and just use default inline values I think.

Reply:
Updated.